### PR TITLE
Added: New cpp Files 

### PR DESCRIPTION
**Pull request by harshalself**
I added some cpp files and included my name is contributers.md

### CODE REVIEW

**Comment by inclinedadarsh** [comment]
Open different pull requests for each file

File: Addition.cpp [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// NAME : HARSHAL PATIL
```
Comment (inclinedadarsh) [inline Addition.cpp:1]: Remove this line

File: Area_Perimeter_Rectangle.cpp [hunk 2]
```diff
@@ -0,0 +1,40 @@
+// NAME : HARSHAL PATIL
```
Comment (inclinedadarsh) [inline Area_Perimeter_Rectangle.cpp:1]: Remove this line

File: Positive_Negative.cpp [hunk 3]
```diff
@@ -0,0 +1,32 @@
+// NAME : HARSHAL PATIL
```
Comment (inclinedadarsh) [inline Positive_Negative.cpp:1]: Remove this line

File: Quotient_Remainder.cpp [hunk 4]
```diff
@@ -0,0 +1,28 @@
+// NAME : HARSHAL PATIL
```
Comment (inclinedadarsh) [inline Quotient_Remainder.cpp:1]: Remove this line

File: Simple_Intrest.cpp [hunk 5]
```diff
@@ -0,0 +1,29 @@
+// NAME : HARSHAL PATIL
```
Comment (inclinedadarsh) [inline Simple_Intrest.cpp:1]: Remove this line

**Review by inclinedadarsh — CHANGES_REQUESTED** [review]

**Comment by harshalself** [comment]
> 

how to do that "different pull request for each file"

**Comment by pranavvp16** [comment]
@harshalself commit with changes in all the file on your forked branch and then Make a pull request

**Comment by harshalself** [comment]
@pranavvp16 i have made those changed in cppcontribute branch .

**Comment by harshalself** [comment]
Hello @inclinedadarsh I have done all changes . Please let me know if there are any remaining. Thank You

**Comment by inclinedadarsh** [comment]
@harshalself after some discussion, we have decided that one person should be able to add only one program so everyone gets an opportunity to make a open-source contribution.

I am going to have to close this PR. You can make a new PR with just one new program.

Also, this programs are really beginner friendly, and are not much complex and this repo is for somewhat complex programs. So please add some complex program from next time. To know more about contributing to the repository, you can check out the [CONTRIBUTING.md](https://github.com/programming-kkw/cpp-important-programs/blob/main/CONTRIBUTING.md) file.

Thank You!

**Comment by harshalself** [comment]
Yeah sure

**Pull request closed without merging**